apps/web-e2e/src/e2e/posts.cy.ts

```diff
@@ -105,7 +105,7 @@ describe('posts', () => {
     latestPostInFeedContentEq(postContent);
   });
 
-  it('can post with emojis', () => {
+  it.only('can post with emojis', () => {
```

Comment:
```suggestion
  it('can post with emojis', () => {
```

You forgot to remove `.only` method.

Reply:
thanks